src/views/mfa-verify/PushForm.js

```diff
@@ -29,6 +31,18 @@ define(['okta'], function (Okta) {
 
     initialize: function () {
       this.enabled = true;
+      this.listenTo(this.options.appState, 'change:isSuccessResponse',
```

Comment:
Couple comments:
- This won't work in the general case, i.e. we are hardcoding the idea that we get a SUCCESS response after the verify state (check out this diagram for state transitions: http://developer.okta.com/assets/img/auth-state-model.png)
- In general if we wanted to do something on success, we'd handle it elsewhere (i.e. this view should only have knowledge about its particular authStatus)

Why not do it the way that rememberDevice does it, i.e. when the user clicks the checkbox?